modules/create.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/named */
+import { todoUl } from '../src/index.js';
+import { deleteTodo, updateTodo } from './crud.js';
+
+const createTodo = (item) => {
+  const itemContainer = document.createElement('li');
+  itemContainer.className = 'todo';
+  itemContainer.draggable = true;
+
+  //   itemContainer.addEventListener('drag', () => {
+  //     console.log('dragged');
+  //   });
+
+  //   itemContainer.addEventListener('dragstart', () => {
+  //     itemContainer.style.opacity = 0.4;
+  //   });
+  //   itemContainer.addEventListener('dragend', () => {
+  //     itemContainer.style.opacity = 1;
+  //   });
+
+  //   itemContainer.addEventListener('dragover', (e) => {
+  //     e.preventDefault();
+  //     return false;
+  //   });
+  //   itemContainer.addEventListener('dragenter', () => {
+  //     itemContainer.classList.add('over');
+  //   });
+  //   itemContainer.addEventListener('dragleave', () => {
+  //     itemContainer.classList.remove('over');
+  //   });
```

Comment:
- [ ] Kindly remove commented-out code. They can make your code messy and more confusing for other developers and don't add functionality.